bundle/phases/deploy.go

```diff
@@ -37,20 +122,16 @@ func Deploy() bundle.Mutator {
 				terraform.Interpolate(),
 				terraform.Write(),
 				terraform.CheckRunningResource(),
-				bundle.Defer(
-					terraform.Apply(),
-					bundle.Seq(
-						terraform.StatePush(),
-						terraform.Load(),
-						metadata.Compute(),
-						metadata.Upload(),
-					),
+				terraform.Plan(terraform.PlanGoal("deploy")),
```

Comment:
Just to double check: if we executed `terraform plan` before, `terraform apply` reuse it or runs terraform plan again? If so, then it might be double as slow

Reply:
Ah, yeah good catch! We indeed are double planning here. Let me send a followup PR to fix that.